README.md

```diff
@@ -31,6 +31,17 @@ The *ArcGIS Runtime Toolkit for iOS* has a *Target SDK* version of *12.0*, meani
 
 ## Instructions
 
+### Swift Package Manager
+
+ 1. Open your project in Xcode
+ 2. Go to *File* > *Swift Packages* > *Add Package Dependency* option 
+ 3. Enter `https://github.com/Esri/arcgis-runtime-toolkit-ios` as the package repository URL
+ 4. Choose version 100.10.0 or a later version. Click Next. Only version 100.10.0 or higher supports Swift Package Manager.
```

Comment:
I tested this in a forked repo and it works ok, I don't think this can be tested here until the SPM PR is merged because the package manifest must be in the root of the repo on the main branch.